### PR TITLE
Fix some incorrect derivative terms in sneut5

### PR DESCRIPTION
**Pull request by yut23**

### CODE REVIEW

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2024-07-28/index.html

File: neutrinos/sneut5.H [hunk 1]
```diff
@@ -686,20 +686,23 @@ void nu_photo(const sneutf_t& sf,
...
-    amrex::Real cc;
+    amrex::Real cc, ccdt;
     amrex::Real c00, c01, c02, c03, c04, c05, c06;
     amrex::Real c10, c11, c12, c13, c14, c15, c16;
     amrex::Real c20, c21, c22, c23, c24, c25, c26;
     amrex::Real dd01, dd02, dd03, dd04, dd05;
     amrex::Real dd11, dd12, dd13, dd14, dd15;
     amrex::Real dd21, dd22, dd23, dd24, dd25;
 
+    amrex::Real taudt = nu_constants::iln10 * sf.tempi;
```
Comment (zingale) [inline neutrinos/sneut5.H:697]: if tau = std::log10(1.e-7 * T), when where is the 1.e-7 in the derivative?

Comment (yut23) [reply neutrinos/sneut5.H:697]: It comes out as a constant term if you rearrange the log: `std::log10(1.e-7 * T) == std::log10(1.e-7) + std::log10(T)` (I was also skeptical at first glance)

Comment (zingale) [reply neutrinos/sneut5.H:697]: ah yes :)

**Review by zingale — APPROVED** [review]

**Pull request merged**